shockbot.py

```diff
@@ -59,73 +57,99 @@
-                        await event.reply('(OpenShock) Next shock available in '+str((next_shock-datetime.now()).seconds)+' seconds.')
-                
-                
-                # Repeated code but for vibrations (TODO: Turn into a function so less code is repeated)
-                if event.raw_text.lower() == '/vibrate':
-                    if datetime.now() > next_vibe:
-                        print('Vibration request received from {} ({})'.format(sender.first_name,event.sender_id))
-                        next_vibe = datetime.now() + timedelta(seconds=VIBE_COOLDOWN)
-                        payload = {'shocks': [{'id': SHOCK_ID, 'type': 'Vibrate', 'intensity': random.randint(VIBE_STR_MIN,VIBE_STR_MAX), 'duration': random.randint(VIBE_DUR_MIN,VIBE_DUR_MAX), 'exclusive': True}], 'customName': 'string'}
-                        response = requests.post(url=url,headers=headers,json=payload)
-                        if response.status_code == 200:
-                            await event.reply('(OpenShock) Vibration sent successfully.')
-                        print(response.content)
-                    else:
-                        print('Vibration request from {} ({}) denied due to cooldown.'.format(sender.first_name,event.sender_id))
-                        await event.reply('(OpenShock) Next vibration available in '+str((next_vibe-datetime.now()).seconds)+' seconds.')
+        # Split args by space to a list
+        cmd = event.raw_text.lower().split(" ")
+
+        # Try to convert the len, dur args to numbers from string, if missing, set them to 0
+        try:
+            cmd[1] = int(cmd[1])
+            cmd[2] = float(cmd[2])
+        except IndexError:
+            cmd.extend([0, 0])
+        # Reply with the usage if the args are not numbers
+        except ValueError:
+            await event.reply(f'The supplied arguments are not numbers. {USAGE}')
+            return
             
-            # User that sent the request was denied, print in console.
-            else:
-                print('Command received from {} ({}) but was ignored due to your settings.'.format(sender.first_name,event.sender_id))
+        # Handle individual commands
+        match cmd[0]:
+            case '/shock':
+                await command(event, cmd)
+            case '/vibrate':
+                await command(event, cmd)
+            case _:
+                await event.reply(f'Unknown command. {USAGE}')
+                print(f'{datetime.now()}: Unknown command sent.')
 
 # Message to let you know the bot is working
 async def welcome_msg():
     me = await client.get_me()
-    print('Python script running as {} ({})'.format(me.first_name,me.id))
+    # First whitelisted ID is treated as the owner IF the blacklist is treated as a whitelist
+    await client.send_message(blacklisted_ids[0], "Ready!")
+    print(f'Python script running as {me.first_name} ({me.id})')
```

Comment:
Should print the user ID before attempting to send messages in case of errors. Move this up a line.